src/script/components/Conversation/Conversation.tsx

```diff
@@ -279,10 +279,6 @@ export const Conversation = ({
   };
 
   const handleMarkdownLinkClick = (event: MouseEvent | KeyboardEvent, messageDetails: MessageDetails) => {
-    if (isAuxRightClickEvent(event)) {
-      // Default browser behavior on right click
-      return true;
-    }
```

Comment:
This check is now done in the parent function (see https://github.com/wireapp/wire-webapp/pull/17640/files#diff-3d29d09ab8471b4309568b6b6e1448ca6de1f9fafbb843c9430201c305bdfe92L328)